webui/react/src/components/FilterForm/components/FilterField.tsx

```diff
@@ -265,7 +265,7 @@ const FilterField = ({
                   value={dayjs(fieldValue).isValid() ? dayjs(fieldValue).utc() : null}
                   onChange={(value: DatePickerProps['value']) => {
                     const dateString = dayjs(value).utc().startOf('date').format();
-                    updateFieldValue(field.id, dateString);
+                    updateFieldValue(field.id, value === null ? null : dateString);
```

Comment:
`DatePickerProps['value']` type is `dayjs.Dayjs | null | undefined`, so i think its safer to handle `undefined` just in case like this

```diff
diff --git a/webui/react/src/components/FilterForm/components/FilterField.tsx b/webui/react/src/components/FilterForm/components/Filt>
index fa45b019a..dc4304fc7 100644
--- a/webui/react/src/components/FilterForm/components/FilterField.tsx
+++ b/webui/react/src/components/FilterForm/components/FilterField.tsx
@@ -264,8 +264,8 @@ const FilterField = ({
                 <DatePicker
                   value={dayjs(fieldValue).isValid() ? dayjs(fieldValue).utc() : null}
                   onChange={(value: DatePickerProps['value']) => {
-                    const dateString = dayjs(value).utc().startOf('date').format();
-                    updateFieldValue(field.id, dateString);
+                    const date = dayjs(value).utc().startOf('date');
+                    updateFieldValue(field.id, date.isValid() ? date.format() : null);
                   }}
                   onOpenChange={setInputOpen}
                 />
```

Reply:
Thanks! This code looks better 🙏 ✅